### PR TITLE
Implement SkyCam component and render it on RunSummary page

### PR DESCRIPTION
**Pull request by dannygoldstein**
responsiveObservingRun

This PR adds a `SkyCam` component and renders it on the `RunSummary` page. The component reads the `skycam_link` property off the backend and renders it as an image on the frontend. The component can be used anywhere, and takes in a JSONified `Telescope` object as input. 

The PR also reskins the OP somewhat. The starlist and current conditions now occupy a single row on large screens, and one row each on smaller screens. See above GIF.

This PR extends #599.

### CODE REVIEW

**Comment by dannygoldstein** [comment]
[ch1051]

**Comment by dannygoldstein** [comment]
@acrellin test added

File: static/js/components/SkyCam.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  const classes = useStyles();
+
+  if (!telescope.skycam_link) {
+    return <div />;
+  }
+
+  const handleImageError = (e) => {
+    e.target.onerror = null;
+    e.target.src =
+      "https://ak.picdn.net/shutterstock/videos/5030309/thumb/1.jpg";
```
Comment (profjsb) [inline static/js/components/SkyCam.jsx:33]: Should not be hardcoding in URLs without a guarantee that they will be there. Suggest downloading and storing this Not Found image locally in the repo. Might as well use an animated gif, right? (https://www.google.com/search?q=static+TV+animated+gif)

Comment (dannygoldstein) [reply static/js/components/SkyCam.jsx:33]: Animated gif will significantly increase the size of the git repo (by ~0.5MB), so keeping to a static image in `static/images/`.

File: data/db_demo.yaml [hunk 2]
```diff
@@ -90,7 +90,7 @@ telescope:
     lon: -155.478
     elevation: 2000
     diameter: 2.56
-    skycam_link: http://catserver.ing.iac.es/weather/archive/concam/concam_labels.png
+    skycam_link: http://kree.ifa.hawaii.edu/allsky/allsky_last.png
```
Comment (stefanv) [inline data/db_demo.yaml:93]: Hit this with every test?

File: skyportal/tests/fixtures.py [hunk 3]
```diff
@@ -216,6 +216,7 @@ class Meta(BaseMeta):
             ),
             nickname=factory.LazyFunction(lambda: f'P200_{uuid.uuid4()}'),
             robotic=False,
+            skycam_link='http://palomarwebcam.com/netcam.jpg',
```
Comment (stefanv) [inline skyportal/tests/fixtures.py:219]: How about this?

File: static/js/components/RunSummary.css [hunk 4]
```diff
@@ -1,6 +1,15 @@
 .source {
 }
 
+.displayInlineBlock {
+  display: inline-block;
+}
+
+.center {
+  margin: auto;
+  padding: 10px;
```
Comment (stefanv) [inline static/js/components/RunSummary.css:10]: Relative

**Comment by stefanv** [comment]
@dannygoldstein Go ahead and merge once you've made those fixes.

File: data/db_demo.yaml (hunk 2 above)
Comment (dannygoldstein) [reply data/db_demo.yaml:93]: fixed

File: static/js/components/RunSummary.css (hunk 4 above)
Comment (dannygoldstein) [reply static/js/components/RunSummary.css:10]: fixed

**Comment by dannygoldstein** [comment]
Merging as GA is green, and the only failure on travis is the unrelated, known flaky test `test_news_feed_prefs_widget`

**Pull request merged**